lib/assessment.rb

```diff
@@ -5,35 +5,52 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
 
 # Question 2
 # Create a new Dog object.
 
-Answer.dog = nil # replace nil with your code
+Answer.dog = Dog.new( "Max", "Lab", "9")
```

Comment:
Age should probably be a number, not a string. But I won't take any points off for it.